CHANGELOG.md

```diff
@@ -62,7 +100,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - console: mark inconsistent remote schemas in the UI (close #5093) (#5181)
 - cli: add missing global flags for seeds command (#5565)
 - docs: add docs page on networking with docker (close #4346) (#4811)
-
+- sdk: add `/contrib` top-level folder and Metadata Types SDK in contrib (#5452)
```

Comment:
There is a conflict in changelog because of other PRs getting merged. When you fix it, could you also remove this line as it has been covered in highlights.